conda/README.md

```diff
@@ -0,0 +1,192 @@
+
+## Searching for Packages
+
+Conda has a search utility that checks all the Channels, but it's important to note that because any type of software
+Can be installed through Conda, that you probably want to search through the details to help identify which ones are meant for Zowe,
+Or use Channels that are distinctly for Zowe so that you can get packages that are strictly for Zowe.
+
+`conda search anything_you_want`
+`conda search --info this_gives_more_details`
+
+## Using Conda with Zowe
+
+Zowe is not yet available in the form of conda packages yet, so it must be installed separately.
+If you have Zowe installed on the same system as Conda, some Zowe plugins installed through Conda will automatically register into Zowe.
+In order to do this, the plugins must be able to find Zowe. You should set environment variables before trying to install the plugins:
+
+### Setting environment variables temporarily:
+z/OS, Linux, Unix:
+```
+export ZOWE_INSTANCE_DIR=/path/to/zowe/instance
+export ZOWE_ROOT_DIR=/path/to/zowe/installation
+```
+
+Windows cmd.exe:
+```
+set ZOWE_INSTANCE_DIR=\path\to\zowe\instance
+set ZOWE_ROOT_DIR=\path\to\zowe\installation
+```
+
+`INSTANCE_DIR` and `ROOT_DIR` are also supported, but the `ZOWE_` prefix helps distinguish its purpose.
+
+### Setting environment variables persistently:
+z/OS, Linux, Unix:
+You can put the `export` statements into the `.profile` file in your home directory to have them apply on login.
+
+Windows:
+There is a UI to set variables, but it varies depending on Windows version.
+Try typing 'environment variable' into the Windows search bar to get to the relevant menu.
+
+### Installing a Zowe Plugin
+
+A Conda package could contain one or more Zowe Plugins, and a Conda package could contain non-Zowe code alongside Zowe Plugins.
+This is left up to the program vendor, and regardless the install process is the same:
+
+`conda install package_name`
+
+If the Zowe environment variables are set, such a package may automatically register plugins into the Zowe instance of your choice.
+
+### Zowe Plugin Configuration
+
+Aside from possible automation during install and uninstall, Conda does not manage Zowe, its configuration, or configuration of the Plugins.
+However, Conda does manage the package files, and therefore you can do additional Zowe tasks on the Plugins by going into the Conda environment.
+Zowe Plugins are intended to be found in a standardized location in the Conda environment,
+
+`/opt/zowe/plugins`
+
+This folder contains plugins, which in turn contain sub-folders that are the Zowe components that they utilize.
+If a plugin uses multiple Zowe components, its contents could be found within multiple component folders.
+
+`/opt/zowe/plugins/my_plugin/app-server`
+`/opt/zowe/plugins/my_plugin/cli`
+
+### Zowe Package Structure
+
+Zowe Plugins packaged into Conda follow the structure outlined here: https://github.com/zowe/zowe-install-packaging/issues/1569
```

Comment:
"You could also view a sample metadata file I made, not to be used as a one size fits all, here: Git link " ?
(because it's not in the issue 1569)